### PR TITLE
[IMP] inventory: refactor WH and locations

### PR DESCRIPTION
**Pull request by Felicious**
Refactor and merge similar warehouse and location docs.
1. Merge redundant info about location types from [Manage Warehouses and Locations](https://www.odoo.com/documentation/15.0/applications/inventory_and_mrp/inventory/management/warehouses/warehouses_locations.html) and [Locations](https://www.odoo.com/documentation/15.0/applications/inventory_and_mrp/inventory/management/warehouses/use_locations.html) 
  Note: 15.0 links are used because the docs will be overwritten once this PR is pushed
2. Merge redundant info about creating and configuring warehouses from [Manage Warehouses and Locations](https://www.odoo.com/documentation/15.0/applications/inventory_and_mrp/inventory/management/warehouses/warehouses_locations.html) and [Create a second warehouse](https://www.odoo.com/documentation/15.0/applications/inventory_and_mrp/inventory/management/warehouses/create_a_second_warehouse.html)
3. Update screenshots to `16.0` and add additional information about new fields, such as **Replenish Location**
4. Rename `warehouses_locations.rst` to `warehouses.rst` and add redirect rule
5. Reorder docs in the `inventory_management` folder, as per @cru-odoo's suggestions :smile: 
6. Add introductory content and term definitions on `inventory_management.rst`, the section for **Inventory Management**.
  Note: This aims to define terms clearly without crowding instructional pages, making it easier for users to find and understand relevant information before diving into detailed instructions.

Tasks:
- [Refactor warehouse and location docs](https://www.odoo.com/odoo/project/3835/tasks/4046637?cids=3): 3 pts
- [Inventory type correction](https://www.odoo.com/odoo/project/3835/tasks/4046637?cids=3) : 2pts

**Important**: in Odoo databases, in Inventory app -> Configuration --> Settings, clicking the "?" icon next to the Storage Locations feature takes you to the Manage Warehouses and Locations documentation, and it should be the Locations documentation instead, after this PR is pushed. Who should I be asking to make this change?
image

### CODE REVIEW

**Comment by Felicious** [comment]
Hi @jero-odoo ! This hefty PR is ready for review! There are several files :see_no_evil:

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst
Comment (jero-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst:35]:
```suggestion
- :guilabel:`Parent Location`: the location within which the new location exists. After the location
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst
Comment (jero-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst:37]:
```suggestion
  describe how a specific location fits within larger areas of the warehouse.
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst
Comment (jero-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst:57]:
```suggestion
  <inventory/warehouses_storage/storage-category>` feature enabled in :menuselection:`Inventory app
  --> Configuration --> Settings`.
```
less than 100 characters

**Review by jero-odoo — APPROVED** [review]
Hey @Felicious great work on this monster of a doc/docs! Just a few things to look at here, other wise it is ready to go!
For a few of the notes in the "example" blocks, there may be times where I was attempting to make grammar suggestions, but I was just having trouble fully understanding what you were trying to describe. If you have any questions let me know. Thanks!

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management.rst
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management.rst:56]: "an area" works!

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management.rst
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management.rst:121]: Thank you for pointing out that this doesn't really make sense to you! I'm going to try to fix the wording, and hopefully, it'll make more sense!

**Comment by Felicious** [comment]
Hi @odoo/us-doc-review ! This is a hefty PR with THREE documents to review :see_no_evil: 
`warehouses.rst`, `use_locations.rst`, and `inventory_management.rst`! Thank you in advance for your extensive work on this!

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst:12]:
```suggestion
section, tick the :guilabel:`Storage Locations` checkbox. Then, click :guilabel:`Save`.
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst:44]:
```suggestion
Additional Information section
```
suggesting this for added specificity

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst:58]:
```suggestion
- :guilabel:`Company`: the company the location belongs to.
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst:59]:
```suggestion
- :guilabel:`Is a Scrap Location?`: tick this checkbox to allow for scrapped/damaged goods to be stored
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst:61]:
```suggestion
- :guilabel:`Is a Return Location?`: tick this checkbox to allow products to be returned to this
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst:60]:
```suggestion
  in this location.
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst:62]:
```suggestion
  location.
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst:63]:
```suggestion
- :guilabel:`Barcode`: used with the *Barcode* app, enter the barcode to :ref:`identify actions
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst:73]:
```suggestion
Cyclic Counting section
```
for added specificity

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst:77]:
```suggestion
(Days)` field to the desired interval. By default, it is set to `0` (no scheduled counts).

For example,
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst:78]:
```suggestion
setting this field to `30`, schedules a count every thirty days. For more specifics on setting up and
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst:83]: Do you think it's vital to mention that these fields are not modifiable? Just a suggestion...

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst:93]:
```suggestion
Logistics section
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst:108]:
```suggestion
Configuration --> Locations`, and select the desired location.
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses.rst:9]:
```suggestion
warehouses, and :doc:`create stock moves <use_routes>` between them.
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses.rst:17]:
```suggestion
Then, select an existing warehouse, or create a new one by clicking :guilabel:`New`. Doing so opens
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses.rst:32]:
```suggestion
- :guilabel:`Warehouse` (*required field*):  the full name of the warehouse.
- :guilabel:`Short Name` (*required field*): the abbreviated code for the warehouse (maximum five
  characters). The short name for the default warehouse in Odoo is `WH`.

  .. important::
     The :guilabel:`Short Name` appears on warehouse documents, so it is recommended to use an
     memorable one, like "WH[first letters of location]" (e.g. `WHA`, `WHB`, etc.).

- :guilabel:`Company` (*required field*): the company that owns the warehouse; this can be set as
  the company that owns the Odoo database, or the company of a customer or vendor.
- :guilabel:`Address` (*required field*): the address of the warehouse. To change the warehouse
  address when creating two or more warehouses, hover over the field, and click the
  :icon:`fa-arrow-right` :guilabel:`(right arrow)`.
```
suggesting putting the 'required field' parenthetical **before** the colon/explanation, as it appears a bit cleaner this way -- totally your call, though

Comment (ksc-odoo) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses.rst:32]: I'm also seeing an "Intrastat region" field here in the runbot I'm using for the review. So, I'm not sure if you want to also mention that (or mention why/how that would/wouldn't appear)... just something to consider during your revisions. 👍 
Screenshot 2024-07-18 131821

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses.rst:50]:
```suggestion
  components from vendors, and dropship them to subcontractors.
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses.rst:52]:
```suggestion
  checkbox to supply subcontractors with raw materials stored in *this* specific warehouse.
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses.rst:53]:
```suggestion
- :guilabel:`Manufacture to Resupply`: tick this checkbox to allow for items to be manufactured in this
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses.rst:59]:
```suggestion
- :guilabel:`Buy to Resupply`: tick this checkbox to allow for purchased products to be delivered to the
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses.rst:62]:
```suggestion
  to pull stock *from* to fulfill orders.
```

File: content/applications/services/field_service/default_warehouse.rst
Comment (ksc-odoo) [inline content/applications/services/field_service/default_warehouse.rst:20]:
```suggestion
feature needs to be activated in the *Inventory* app. It is also necessary to have more than one
```

File: content/applications/services/field_service/default_warehouse.rst
Comment (ksc-odoo) [inline content/applications/services/field_service/default_warehouse.rst:24]:
```suggestion
Either set it up :ref:`for the user profile <default-warehouse/my-profile>`, or :ref:`for all
users <default-warehouse/all-users>`.
```

**Review by ksc-odoo — APPROVED** [review]
Hey @Felicious -- just finished my Final Review of this "bulky" PR. Great job! Very in-depth, yet easy-to-follow. I have a handful of comments/feedback/etc. sprinkled throughout the multiple docs in this PR, but I don't anticipate you having any issues, so I am approving now. Once you implement the necessary adjustments, feel free to move this to Tech Review. Thanks! 👍

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst:5]: While I appreciate your feedback to improve the removal strategy doc, to keep this PR focused on warehouses and locations, I'll be implementing these suggestions in a future PR (:

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/use_locations.rst:83]: Do you think it's implied by the word "displayed"?

File: content/applications/services/field_service/default_warehouse.rst
Comment (Felicious) [reply content/applications/services/field_service/default_warehouse.rst:20]: Same as above-- since this is a field service doc, I plan to not make any content changes, and just wanted to update a link that wouldve been broken upon publishing this PR.

**Comment by Felicious** [comment]
Thank you for your thorough and very helpful review, @ksc-odoo !
Requesting a technical review from @StraubCreative :smile: This is a very long PR with 3 docs :see_no_evil: All my notes are in the PR description!

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management.rst:8]: This doc is well-written and organized. Everything is clearly explained, and the structure makes sense. Nice job 👍

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management.rst:85]: this screenshot is a bit hard to read 
consider making it larger and zooming in if you can.

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management.rst:96]: same here, this is hard to read
can we zoom in on the UI more here (or reduce the browser width a little bit, before mobile breakpoint)?

**Review by StraubCreative — APPROVED** [review]
Looks good to me @Felicious, nicely done 👍 
My only feedback is to size up the screenshots a little bit and make sure they're in HD, as they're a bit difficult to read. Can do that on another PR if you like.
.....
@robodoo delegate=Felicious

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management.rst
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management.rst:85]: done!

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management.rst
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management.rst:96]: got it, zoomed (:

**Comment by Felicious** [comment]
@robodoo r+

**Pull request closed without merging**
